orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/model/PluginArtifact.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.front50.model;
+
+import java.util.List;
+import java.util.regex.Pattern;
+import javax.annotation.Nonnull;
+import lombok.Data;
+
```

Comment:
Can we move these model/DTO objects to kork-plugins-api ?

Reply:
I think that's a broader issue - we do this type of model duplication across many services in Spinnaker.  I don't think we have a great story around sharing these.  I don't think I want `kork-plugins-api` to be a dependency in `orca-front50` since `kork-plugins-api` is really the library for writing plugins/extension points, and I'm not sure I want to break away from current patterns at the moment.

Reply:
`kork-plugins-api` is for the API contract between actual plugins and Spinnaker. Actual plugins won't need these models.

Reply:
If we're thinking about shared models, I think that needs a separate discussion, as there's a lot of paths forward here and we'd want consensus on direction before doing anything.